### PR TITLE
README: add section on how to integrate with fuse

### PR DESCRIPTION
**Pull request by kusma**
This adds a short section about how to configure fuse/uno to always pick up the checked-out fuselibs.

I'm not in love with this, but we need this information in some form. Some feedback would be greatly appreciated.

One of the big lacks, is that this doesn't really mention where to put the unoconfig-file, might be confusing to people who aren't used to dealing with unoconfig-files (which is probably most users). But I don't really know what the best advice would be.

We used to have something like this at the root of the repo, which made things work when using `Stuff/uno` from inside the fuselibs-repo. But that was removed by @mortend when the package-manager was introduced. We could consider adding it back...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,18 @@ the UI framework used in [Fuse](https://www.fusetools.com/) apps.
...
+
+### Fuse
+
+If you want to use a self-built fuselibs installation with an installed Fuse, you can do this through a `.unoconfig`-file, something like this:
+
+```
+Packages.SourcePaths += <path-to-fuselibs>/Source
+```
+
+You (obviously) need to replace `<path-to-fuselibs>` above with the actual path to your fuselibs checkout.
```
Comment (sebbert) [inline README.md]: The sarcasm here is (obviously) not needed 😉

File: README.md [hunk 2]
```diff
@@ -19,6 +19,18 @@ the UI framework used in [Fuse](https://www.fusetools.com/) apps.
 * `build.sh` runs stuff (which downloads uno if needed), and builds all packages.
 * `test.sh` runs all tests.
 
+
+### Fuse
+
+If you want to use a self-built fuselibs installation with an installed Fuse, you can do this through a `.unoconfig`-file, something like this:
```
Comment (sebbert) [inline README.md]: This doesn't mention where to place the .unoconfig file. For instance:

If you want to use a locally built copy of fuselibs with an installed Fuse, create a file named `.unoconfig` in your home directory and add the following:

File: README.md (hunk 1 above)
Comment (kusma) [inline README.md]: Hehe, good point ;)

File: README.md (hunk 2 above)
Comment (kusma) [inline README.md]: @sebbert: Yeah, I wrote some rant about that in the cover-letter of this PR (above). I'm still a little bit torn if we should advice people about the home-directory, as that affects all projects. We could also advice them to put the `.unoconfig` in their project folder. And finally, we could just reinstate the top-level .unoconfig in the fuselibs repo, and tell people to run uno from inside there.

I'm not sure which option is better.

File: README.md (hunk 1 above)
Comment (mortoray) [inline README.md]: Don't you need to somehow remove the old fuselibs path? If you simply do `+=` don't you get the checked out fuselibs and the installed fuselibs? I don't know which is resolved in that case.

Comment (kusma) [inline README.md]: @mortoray: I believe fuse doesn't use Packages.**Source**Path, and I believe uno prefers using the source-path. This is reported to work for users...

File: README.md (hunk 2 above)
Comment (sebbert) [inline README.md]: @kusma Oh sorry, didn't see that – skipped right to the diff on this one.

I think we should document both options, as they can both be useful in different cases. We can always add a sentence about running uno from the repo directory if we decide to reinstate the repo-level .unoconfig.

How about something like this?

---

You may use a locally built copy of fuselibs with an installed Fuse. This is done by creating a file named `.unoconfig` in either a Fuse project directory (applies to that project only), or in your home directory (applies to all projects). It should contain something like the following:

Comment (kusma) [inline README.md]: @sebbert: Yeah, that sounds good. I was afraid of this quickly turning into a `.unoconfig`-tutorial, but your suggestion balances this nicely.

**Comment by kusma** [comment]
@sebbert: Pushed fixes for your input :)

**Pull request merged**